app/src/main/res/values/colors.xml

```diff
@@ -11,6 +11,7 @@
     <color name="habit_list_text">#68B0AB</color>
     <color name="theme_dark_grey">#696D7D</color>
     <color name="habit_add_button">#C8D5B9</color>
+    <color name="habit_day_selected">#8FC0A9</color>
```

Comment:
Added the colour without realizing that the colour was already included in the palette, kept it in for readability though.